src/service/BookingService.java

```diff
@@ -30,6 +31,22 @@ public BookingService(BookingDataDAO bookingDataDAO,GuestDataDAO guestDataDAO) {
 	}
 
 	public Integer saveBooking(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethodDTO methodPayment)  {
+		if (entryDate == null) {
+			throw new IllegalArgumentException("entryDate was null");
+		}
+		if (departureDate == null) {
+			throw new IllegalArgumentException("departure was null");
+		}
+		if (methodPayment == null) {
+			throw new IllegalArgumentException("method payment was null");
+		}
+		if (entryDate.isBefore(LocalDateTime.now())) {
+			throw new IllegalArgumentException("entry date is incorrect");
+		}
+		if (departureDate.isBefore(entryDate)) {
+			throw new IllegalArgumentException("departure date is incorrect");
```

Comment:
Please use a more accurate message such as "Departure date can't be before entry date.".